extension-manifest-v2/src/utils/api.js

```diff
@@ -33,22 +33,27 @@ class Api {
 			return this._refreshPromise;
 		}
 
-		this._refreshPromise = fetch(`${this.config.AUTH_SERVER}/api/v2/refresh_token`, {
-			method: 'POST',
-			credentials: 'omit',
-		}).then((res) => {
-			if (res.status > 400) {
-				throw res.json();
+		const pending = (async () => {
+			const response = await fetch(`${this.config.AUTH_SERVER}/api/v2/refresh_token`, {
+				method: 'POST',
+				credentials: 'omit',
```

Comment:
this request is lacking any credentials, it cannot work. lets try to fix it in this PR

Reply:
The experimental authentication server API will try now first the HTTP headers `UserId` and `RefreshToken` and only then fallback to cookies.